frontend/src/components/widgets/BaseButtonLink.js

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import Tab from "@material/react-tab";
```

Comment:
Please don't edit this. We use Buttons all over the place.

The issue says: Dont use buttons in the app bar, use tabs instead in the app bar.

The issue does not say: Never use buttons anywhere ever again. We are jkust changing one little part of the application so it works differently.